src/pages/ProblemEditor/ProblemEditorContainer/ExampleValidator.tsx

```diff
@@ -1,15 +1,77 @@
 import { Box, FormControlLabel, FormGroup, Switch } from "@mui/material";
 import React from "react";
+import { useDispatch } from "react-redux";
+import { useAppSelector } from "../../../app/common/hooks";
+import {
+  validateCode,
+  validateMetadata,
+  validateProblem,
+  validateServerConfig,
+  validateTestEditor,
+} from "./problemEditorContainerSlice";
 
-interface Props {
-  handleValidation: (isValid: boolean) => void;
-  validationStatus: () => boolean;
-}
+export const ExampleValidator = () => {
```

Comment:
Why is it called ExampleValidator? seems like something left behind from a tutorial